### ISSUE TITLE
Can't use "None" as _id for aggregations

### ISSUE THREAD

**Issue by ymost** [mongomock/mongomock#204]
The following code runs fine with pymongo:

```
client = pymongo.MongoClient()
col = client['test_db']['test_col']
col.aggregate([{'$group': {'_id': None, 'sum': {'$sum': '$f'}}}])
```

With mongomock you get:

```
TypeError: itemgetter expected 1 arguments, got 0
```

**Comment by polosatyi** [comment]
+ 1

**Comment by drorasaf** [comment]
I Have just create a PR for this #256

**Comment by pcorpet** [comment]
Fixed by #256.
